content/_data/changelogs/weekly.yml

```diff
@@ -6727,6 +6727,124 @@
     message: |-
       Don't lose SCM configuration when saving job (regression in 2.224).
 
+- version: 2.226
+  date: 2020-03-14
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4568
+    issue: 61429
+    authors:
+    - fqueiruga
+    message: |-
+      Fix drag & drop for previously saved steps in the job configuration form (regression in 2.217).
+  - type: major rfe
+    category: major rfe
+    pull: 4546
+    authors:
+    - daniel-beck
+    message: |-
+      Organize entries on the Manage Jenkins page into categories and show them in a grid.
+  - type: rfe
+    category: rfe
+    pull: 4555
+    issue: 61224
+    authors:
+    - romenrg
+    message: |-
+      Remove the unnecessary "monitor[s]" text next to the bell for a cleaner UI.
+      Change the colors of the notifications next to the bell to make them more noticeable.
+  - type: rfe
+    category: rfe
+    pull: 4549
+    issue: 54662
+    authors:
+    - timja
+    message: |-
+      Allow usage statistics to be configured with the configuration-as-code plugin.
+  - type: rfe
+    category: rfe
+    pull: 4563
+    authors:
+    - timja
+    references:
+      - pull: 4563
+      - url: https://github.com/jenkinsci/ssh-cli-auth-module/releases/tag/ssh-cli-auth-1.8
+        title: ssh-cli-auth 1.8 changelog
+    message: |-
+      Allow ssh authorized keys to be configured with the configuration-as-code plugin.
+  - type: rfe
+    category: rfe
+    pull: 4548
+    issue: 60921
+    authors:
+    - fqueiruga
+    message: |-
+      Use modern system fonts provided by the browser when possible.
+      Changes font size for body copy and headings to improve consistency and legibility.
+  - type: rfe
+    category: rfe
+    pull: 4561
+    authors:
+    - daniel-beck
+    message: |-
+      Update bundled Script Security Plugin from 1.70 to 1.71.
```

Comment:
Link changelog?

Reply:
Thanks for the suggestion1  See 5ce13778a277ecb48ecf7fd34416a0201d0a7f59